plugins/uefi/README.md

```diff
@@ -8,15 +8,6 @@ The Unified Extensible Firmware Interface (UEFI) is a specification that
 defines the software interface between an OS and platform firmware.
 With the UpdateCapsule boot service it can be used to update system firmware.
 
-Build Requirements
-------------------
-
-For UEFI capsule support, you need to install fwupdate 0.5 or later.
-* source:		https://github.com/rhinstaller/fwupdate
-* rpms:			https://pjones.fedorapeople.org/fwupdate/
-* debs (Debian):	https://tracker.debian.org/pkg/fwupdate
-* debs (Ubuntu):	https://launchpad.net/ubuntu/+source/fwupdate
-
 If you don't want or need this functionality you can use the
 `--disable-uefi` option.
```

Comment:
This should be `-Dplugin_uefi=false` now

Reply:
Fixed, thanks.